src/implementations/twig/components/page-banner/page-banner.html.twig

```diff
@@ -58,6 +60,9 @@
 <section class="{{ _css_class }}"{{ _extra_attributes|raw }}>
 {% if _image is not empty and _variant in ['image', 'image-shade', 'image-gradient'] %}
   <div class="ecl-page-banner__image" style="background-image:url({{ _image }})"></div>
+{% endif %}
+{% if _copyright is not empty and _variant in ['image', 'image-shade', 'image-gradient'] %}
```

Comment:
I wonder if this is the right approach. i mean that i would imagine a use case where the copyright is needed but no text would be associated to it, not sure if it's a valid use case but in this case we should probably have a boolean to control the display and a text parameter to set an optional text, but you know what the requirements are better than me..